style.css

```diff
@@ -0,0 +1,848 @@
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+#feature-hr {
+  margin-bottom: 10px;
+}
+
+.first {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+}
+
+.second {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+}
+
+.third {
+  transform: translateY(100%);
+  transition: 0.7s;
+  display: none;
+}
+
+.third.active {
+  display: flex;
+  flex-direction: row;
+  transform: translateY(0%);
+  list-style: none;
+}
+
+.fourth {
+  transform: translateY(100%);
+  transition: 0.7s;
+  display: none;
+}
+
+.fourth.active {
+  display: flex;
+  flex-direction: row;
+  transform: translateY(0%);
+  list-style: none;
+}
+
+.fifth {
+  transform: translateY(100%);
+  transition: 0.7s;
+  display: none;
+}
+
+.fifth.active {
+  display: flex;
+  flex-direction: row;
+  transform: translateY(0%);
+  list-style: none;
+}
+
+.sixth {
+  transform: translateY(100%);
+  transition: 0.7s;
+  display: none;
+}
+
+.sixth.active {
+  display: flex;
+  flex-direction: row;
+  transform: translateY(0%);
+  list-style: none;
+}
+
+.li-items {
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+}
+
+.anime-sub {
+  color: #ec5242;
+  font-size: 15px;
+}
+
+.anime-name {
+  font-size: 20px;
+  color: #272a31;
+}
+
+.animedes {
+  font-size: 10px;
+  color: #272a31;
+}
+
+.an1 {
+  width: 100px;
+  height: 100px;
+}
+
+.more {
+  justify-content: center;
+  position: relative;
+  width: 300px;
+  height: 75px;
+  transform: translateY(0%);
+  transition: 0.7s;
+  color: #ec5242;
+}
+
+.more.active {
+  transform: translateY(100%);
+}
+
+#about-head {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 15px;
+}
+
+#about-intro {
+  font-size: 25px;
+  color: #ec5242;
+}
+
+#about-intro-heading {
+  font-size: 40px;
+  color: #ec5242;
+  margin-bottom: 30px;
+}
+
+#about-cont {
+  font-size: 20px;
+  color: #272a31;
+  background-color: #d3d3d3;
+  border: 1px solid gray;
+  justify-content: center;
+  text-align: center;
+}
+
+#about-con {
+  font-size: 15px;
+  color: #272a31;
+}
+
+#con-em {
+  font-size: 15px;
+  color: black;
+}
+
+#logo {
+  width: 450px;
+  height: 200px;
+}
+
+hr {
+  border: 0;
+  clear: both;
+  display: block;
+  background-color: #ec5242;
+  height: 1px;
+  width: 40px;
+}
+
+#logo-part {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-bottom: 1px black;
+}
+
+#meetup-logo {
+  margin-bottom: 20px;
+}
+
+#meetup-des {
+  margin-top: 10px;
+}
+
+#featured-animes {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#feature-heading {
+  margin-top: 30px;
+  margin-bottom: 20px;
+}
+
+#feature-des {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+#feature1 {
+  width: 250px;
+  height: 250px;
+}
+
+#feature2 {
+  width: 250px;
+  height: 250px;
+}
+
+#featured-images {
+  background-color: rgba(236, 82, 66, 0.8);
+}
```

Comment:
- [ ] Great work on making these images vertical in mobile display, but images are not centered as in the design template. You can use margins or text-align to align content